File: nga/app/src/main/java/org/gosky/nga/view/MainView.java
package org.gosky.nga.view;

import com.kungfu.dbflow.History;

import org.gosky.base.mvp.BaseView;
import org.gosky.nga.data.entity.BoardBean;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by guozhong on 16/10/24.
 */

public interface MainView extends BaseView {
    void showBoard(List<BoardBean.ResultBean.GroupsBean> mList);
    void showHistory(List<History> list);
}

File: nga/app/src/main/java/org/gosky/nga/view/MainView.kt
package org.gosky.nga.view

import org.gosky.base.mvp.BaseView
import org.gosky.nga.data.entity.BoardBean

/**
 * Created by guozhong on 16/10/24.
 */

interface MainView : BaseView {
    fun showBoard(mList: MutableList<BoardBean.ResultBean.GroupsBean>)
}
